pook/interceptors/__init__.py

```diff
@@ -24,14 +24,14 @@
     interceptors.append(AIOHTTPInterceptor)
 
 
-def add(*interceptors):
+def add(*custom_interceptors):
     """
     Registers a new HTTP client interceptor.
 
     Arguments:
-        *interceptors (interceptor): interceptor(s) to be added.
+        *custom_interceptors (interceptor): interceptor(s) to be added.
     """
-    interceptors.append(*interceptors)
+    interceptors.append(*custom_interceptors)
```

Comment:
Dammit! This is the kind of tricky silly issue you expect the linter will catch up for you, but it's hard even for a linter.